sdk/trace/src/test/java/io/opentelemetry/sdk/trace/testbed/actorpropagation/ActorPropagationTest.java

```diff
@@ -47,7 +46,7 @@ void before() {
   void testActorTell() {
     try (Actor actor = new Actor(tracer, phaser)) {
       phaser.register();
-      Span parent = tracer.spanBuilder("actorTell").setSpanKind(Kind.PRODUCER).startSpan();
+      Span parent = tracer.spanBuilder("actorTell").setSpanKind(Span.Kind.PRODUCER).startSpan();
```

Comment:
It took error prone being annoying to notice we forgot to move Kind to top level. Damn, oh well. This new error prone check did validate for me that inner classes cause reasonable grief.

Reply:
;(